client/my-sites/sidebar-unified/menu.jsx

```diff
@@ -60,38 +60,40 @@ export const MySitesSidebarUnifiedMenu = ( {
 	}, [ selected, childIsSelected, reduxDispatch, sectionId, sidebarCollapsed ] );
 
 	return (
-		<ExpandableSidebarMenu
-			onClick={ () => {
-				if ( link ) {
-					if ( isExternal( link ) ) {
-						// If the URL is external, page() will fail to replace state between different domains
-						externalRedirect( link );
-						return;
+		<li>
+			<ExpandableSidebarMenu
+				onClick={ () => {
+					if ( link ) {
+						if ( isExternal( link ) ) {
+							// If the URL is external, page() will fail to replace state between different domains
```

Comment:
Lets end comments in periods, see https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#single-line-comments

Reply:
Fixed 7f19464